app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -526,4 +529,22 @@ class UploadWorker(var appContext: Context, workerParams: WorkerParameters) :
             curentNotification.build()
         )
     }
+
+    /**
+     * Method used to get Pending intent for opening different screen after clicking on notification
+     * @param toClass
+     * @param params
+     */
+    private fun getPendingIntent(toClass:Class<out BaseActivity> ,params:ArrayList<String>? = null):PendingIntent
+    {
+        val intent=Intent(appContext,toClass)
+        params.let {
+            intent.putStringArrayListExtra("params",params)
```

Comment:
Is this needed? also even if it is,  extract this as a constant, I am assuming this is for someone to use eventually when this is clicked?

Reply:
Yeah actually i created function to getPending intent so even in future someone wants to use it for another notification he or she can use that one.

Reply:
What I meant is about the hardcoded string "params", that can be extracted out. Also, where are we using the params?

Reply:
> What I meant is about the hardcoded string "params", that can be extracted out. Also, where are we using the params, since it's anyways null?

Yeah actually it's not needed now, just created for future need